core/src/iam/signup.rs

```diff
@@ -20,143 +22,141 @@ pub async fn signup(
 	// Parse the specified variables
 	let ns = vars.get("NS").or_else(|| vars.get("ns"));
 	let db = vars.get("DB").or_else(|| vars.get("db"));
-	let sc = vars.get("SC").or_else(|| vars.get("sc"));
+	let ac = vars.get("AC").or_else(|| vars.get("ac"));
 	// Check if the parameters exist
-	match (ns, db, sc) {
-		(Some(ns), Some(db), Some(sc)) => {
+	match (ns, db, ac) {
+		(Some(ns), Some(db), Some(ac)) => {
 			// Process the provided values
 			let ns = ns.to_raw_string();
 			let db = db.to_raw_string();
-			let sc = sc.to_raw_string();
-			// Attempt to signup to specified scope
-			super::signup::sc(kvs, session, ns, db, sc, vars).await
+			let ac = ac.to_raw_string();
+			// Attempt to signup using specified access method
+			// Currently, signup is only supported at the database level
+			super::signup::db(kvs, session, ns, db, ac, vars).await
 		}
 		_ => Err(Error::InvalidSignup),
 	}
 }
 
-pub async fn sc(
+pub async fn db(
```

Comment:
Same here, should this be called something more descriptive like `db_record`?

Reply:
I agree. I have changed it to `db_access` in [c6d0d06](https://github.com/surrealdb/surrealdb/pull/3988/commits/c6d0d062abcdda4e9378ec53a21975769163f265). Although we have no plans for `signup` to create anything other than records, I prefer to keep it generic and in sync with `signin` to support future authentication methods that may support signing up.